app/src/main/java/com/alif/newsreader/activities/MainActivity.java

```diff
@@ -124,6 +128,10 @@ protected void onCreate(Bundle savedInstanceState) {
         // The specified network connection is not available. Displays error message in webview.
         errorMsgWebView = (WebView) findViewById(R.id.webview);
         errorMsgWebView.setVisibility(View.GONE);
+
+        mAdView = (AdView) findViewById(R.id.adView);
+        AdRequest adRequest = new AdRequest.Builder().addTestDevice("118F106CA47E3799F5997AFCAFADEFCF").build();
```

Comment:
can you please keep this in Constant.java file and pick from there? as **AdMobTestDeviceId**